packages/neo-one-node-bin/src/bin/neo-one-node.ts

```diff
@@ -0,0 +1,19 @@
+import { FullNode } from '@neo-one/node';
+// tslint:disable-next-line:no-implicit-dependencies
+import { createEnvironment, createMonitor, createOptionsStream, createSettings } from '@neo-one/node-bin';
+
+const monitor = createMonitor({ logConsole: true });
```

Comment:
I would move all of this into class similar to `FullNode` that delegates to `FullNode`. Then this file is just `new Node(<options to fill in with the rc module later>).start()`.